pymeasure/instruments/fluke/fluke7341.py

```diff
@@ -0,0 +1,68 @@
+#
+# This file is part of the PyMeasure package.
+#
+# Copyright (c) 2013-2021 PyMeasure Developers
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+#
+
+from pymeasure.instruments import Instrument
+from pymeasure.instruments.validators import strict_discrete_set, strict_range
+from pymeasure.adapters import SerialAdapter
+
+class FlukeSerialAdapter(SerialAdapter):
+    def write(self, command):
+        super().write(command + "\r\n")
+
+class Fluke7341(Instrument):
+    """ Represents the compact constant temperature bath from Fluke
+    """
+
+    set_point = Instrument.control("s", "s=%g",
+                                   """ Set the bath temperature. This property can be read
+                                   """,
+                                   validator=strict_range,
+                                   values = (-40, 150),
+                                   get_process=lambda x: float(x.split()[1])
```

Comment:
As almost all of the properties have the same `get_process`, maybe it would be nice (in the DRY-mindset) to define a (static)method that performs the split; if you pass it as the `preprocess_reply` rather than the `get_process` keyword argument, you can even leave out the casting in the method, as the casting to float is attempted by default after the `preprocess_reply`

Reply:
You mean using: `preprocess_reply=lambda x: x.split()[1]`. I wouldn't go for a static method since the `lambda` is almost as compact a method name. What do you think ?

Reply:
Yeah like this is also fine with me indeed